paddle/framework/block_desc.cc

```diff
@@ -55,6 +55,12 @@ OpDescBind *BlockDescBind::AppendOp() {
   return ops_.back().get();
 }
 
+void BlockDescBind::AppendAllocatedOp(std::unique_ptr<OpDescBind> &op_desc) {
```

Comment:
I think it is pretty hard to figure out the `op_desc` has been changed inside `AppendAllocatedOp` for others.

It is better to use `std::move` outside, and accept the `rvalue reference` here.

Reply:
Good idea!